tests/integration-tests/vpc_builder.py

```diff
@@ -64,64 +63,116 @@ class VPCConfig(NamedTuple):
 class VPCTemplateBuilder:
     """Build troposphere CFN templates for VPC creation."""
 
-    def __init__(self, vpc_config, description="VPC built by VPCBuilder"):
+    def __init__(
```

Comment:
If you want to move things outside init, we can either add them as parameter to build or leave them there. That's because moving things outside as you suggested makes the code much harder to read (in my opinion). Anyway, I have refactored that a bit to make it more readable. Let me know if further refactorization is needed (like moving the parameters to build)